BookingsApi/BookingsApi.UnitTests/Controllers/JudiciaryPersonControllerTests.cs

```diff
@@ -0,0 +1,278 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Controllers;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Controllers
+{
+    public class JudiciaryPersonControllerTests
+    {
+        protected JudiciaryPersonController Controller;
+        protected Mock<IQueryHandler> QueryHandlerMock;
+        protected Mock<ICommandHandler> CommandHandlerMock;
+        protected Mock<ILogger<JudiciaryPersonController>> LoggerMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            QueryHandlerMock = new Mock<IQueryHandler>();
+            CommandHandlerMock = new Mock<ICommandHandler>();
+            LoggerMock = new Mock<ILogger<JudiciaryPersonController>>();
+            Controller = new JudiciaryPersonController(QueryHandlerMock.Object, CommandHandlerMock.Object, LoggerMock.Object);
+        }
+
+        [Test]
+        public async Task Should_return_ok_result_with_empty_response_for_zero_bulk_inserts()
+        {
+            var request = new List<JudiciaryPersonRequest>();
+
+            var result = await Controller.BulkJudiciaryPersonsAsync(request);
+
+            result.Should().NotBeNull();
+            var objectResult = result as ObjectResult;
```

Comment:
safer to assert the type than assume
`result.Should().BeOfType<ObjectResult>();`

Reply:
I can't, I need to use the cast as object later on

Reply:
You can still keep the remainder of the code but before line 44, if you assert on the returned type it safely cast to ObjectResult


`result.Should().NotBeNull();`
`result.Should().BeOfType<ObjectResult>();` // failing here would return a better failure reason than an exception being thrown
`var objectResult = result as ObjectResult;`
`objectResult.Should().NotBeNull();``

Reply:
you can still keep the the remainder of the code but before line 44, if you assert on the returned type it will return a better failure reason than throwing an exception. 

`result.Should().BeOfType<ObjectResult>();`

confirms the result is not null and is of type objecteresult before the rest of the test continues

Reply:
replace the null chek with the ofType check, it does the null check but also type check and give a nice error :)

Reply:
Yes but I need the cased object later to check statuscode

Reply:
simply suggesting the replacement of the null assertion with the oftype assertion, shouldnt effect the rest of the code?

Reply:
i wont block PR for this change but i feel its a solid improvement